packages/default/scss/notification/_layout.scss

```diff
@@ -1,16 +1,15 @@
 @include exports("notification/layout") {
 
-    $alert-padding-y: 1em !default;
-    $alert-padding-x: 1em !default;
-
     // Base
     .k-notification {
-        padding: 0;
         @include border-radius();
+        padding: 0;
     }
     .k-notification-wrap {
+        padding: $notification-padding-y calc(#{$notification-padding-x} + #{$icon-size} + #{$icon-spacing}) $notification-padding-y $notification-padding-x;
```

Comment:
the calc thing can be extracted as a local variable, say `$notification-padding-end` and be used in both directions